### PR TITLE
Add Movie Seat Booking

### PR DESCRIPTION
**Pull request by VBystrov**
### html-movie-seat-booking

[demo](https://vbystrov.github.io/html-movie-seat-booking/index.html)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/VBystrov/html-movie-seat-booking/css/style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+body {
+  font-family: 'Noto Sans', sans-serif;
+  font-size: 16px;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
```
Comment (A-Ostrovnyy) [inline submissions/VBystrov/html-movie-seat-booking/css/style.css]: Fixed height not good practice in the blocks with various content. Use `min-height` instead. 

image

File: submissions/VBystrov/html-movie-seat-booking/css/style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  margin: 0 10px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.seat-place {
```
Comment (A-Ostrovnyy) [inline submissions/VBystrov/html-movie-seat-booking/css/style.css:59]: Add `cursor: pointer` for better identification interactive elements.

File: submissions/VBystrov/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+                <div class="seat-number">9</div>
+              </label>
+            </div>
+          </div>
+          <div class="legend">
+            <div>
+              <p>Free</p>
+              <div class="row">
+                <label class="seat-place">
+                  <input
```
Comment (A-Ostrovnyy) [inline submissions/VBystrov/html-movie-seat-booking/index.html]: Functionally it's just info elements, no need to use interactive elements for it.

File: submissions/VBystrov/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+                    type="checkbox"
+                    class="seat-checkbox"
+                    value="test"
+                    onclick="return false;"
```
Comment (A-Ostrovnyy) [inline submissions/VBystrov/html-movie-seat-booking/index.html]: Avoid using `onclick=` attribute it's a bad practice, but in some cases, it's the only solution(not this time).

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below!

File: submissions/VBystrov/html-movie-seat-booking/css/style.css (hunk 1 above)
Comment (VBystrov) [reply submissions/VBystrov/html-movie-seat-booking/css/style.css]: Done.

File: submissions/VBystrov/html-movie-seat-booking/css/style.css (hunk 2 above)
Comment (VBystrov) [reply submissions/VBystrov/html-movie-seat-booking/css/style.css:59]: Done.

File: submissions/VBystrov/html-movie-seat-booking/index.html (hunk 3 above)
Comment (VBystrov) [reply submissions/VBystrov/html-movie-seat-booking/index.html]: Input removed.

File: submissions/VBystrov/html-movie-seat-booking/index.html (hunk 4 above)
Comment (VBystrov) [reply submissions/VBystrov/html-movie-seat-booking/index.html]: Onclick removed with input.

File: submissions/VBystrov/html-movie-seat-booking/css/style.css (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/VBystrov/html-movie-seat-booking/css/style.css:59]: No need to add `cursor: pointer` for info seats.

image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!
One small issue)

File: submissions/VBystrov/html-movie-seat-booking/css/style.css (hunk 2 above)
Comment (VBystrov) [reply submissions/VBystrov/html-movie-seat-booking/css/style.css:59]: Added style for info seats with default cursor.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**